core/lib/libtomcrypt/include/tomcrypt_pk.h

```diff
@@ -288,10 +288,18 @@ int  ecc_decrypt_key(const unsigned char *in,  unsigned long  inlen,
                            unsigned char *out, unsigned long *outlen, 
                            ecc_key *key);
 
+int ecc_sign_hash_raw(const unsigned char *in,  unsigned long inlen,
+                            void   *r,   void *s,
+                            prng_state *prng, int wprng, ecc_key *key);
+
```

Comment:
Yes, I know. I tried to keep the spacing rules (if any) there is on Libtomcrypt, such as dsa_sign_hash_raw(). Let me know if I should use the OPTEE coding rules for this and below remark.